mobile/library/cc/log_level.h

```diff
@@ -7,7 +7,15 @@
 namespace Envoy {
 namespace Platform {
 
-using LogLevel = Envoy::Logger::Logger::Levels;
+enum class LogLevel {
```

Comment:
I guess we can now remove the IWYU pragma since we're not expecting callers to use it directly?